tests/run_tests.py

```diff
@@ -45,6 +121,9 @@
     argparser.add_argument('-j', '--jenkins', dest='jenkins',
                            help='run also tests that should run only in a CI environment',
                            action='store_true')
+    argparser.add_argument('-c', '--core', dest='core',
+                           help='run only tests that cover basic functionality of the library',
```

Comment:
And regression tests, right?

Reply:
Yes. I'll probably change the command line options to allow more detailed selection, but right now it runs regression tests too. Fixed.